scripts/download_incr_DB.py

```diff
@@ -301,6 +312,26 @@ def calculate_multipart_etag(source_path, chunk_size):
 	return new_etag
 				
 def run():
+	dir_name = STORAGE_PATH + "/historical-data"
+	main_persistence = STORAGE_PATH + "/persistence"
+	try:
+		if(Exclude_microBlocks == False and Exclude_txnBodies == False):
+			if os.path.exists(dir_name) and os.path.isdir(dir_name):
+				if not os.listdir(dir_name):
+					# download the static db
+					print("Dowloading static historical-data")
+					download_static_DB.start(STORAGE_PATH)
+				else:
+					print("Already have historical blockchain-data!. Skip downloading again!")
+			else:
+				# download the static db
+				print("Dowloading static historical-data")
+				download_static_DB.start(STORAGE_PATH)
+	except Exception as e:
+		print(e)
+		print("Failed to download static historical-data!")
```

Comment:
I think it's better to comine this with the exception instead of two separate print calls.